tests/e2e/e2e_setup_test.go

```diff
@@ -24,24 +24,48 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
```

Comment:
maybe rename this to `sdkgovtypes` and just have `govtypes` below

Reply:
Changed the package name on latest commit